gremlin-driver/src/test/java/org/apache/tinkerpop/gremlin/driver/TestWSGremlinInitializer.java

```diff
@@ -136,5 +145,28 @@ private String returnSingleVertexResponse(final UUID requestID) throws Serializa
 
             return SERIALIZER.serializeResponseAsString(ResponseMessage.build(requestID).result(t).create());
         }
+
+        private String returnSimpleStringResponse(final UUID requestID, String message) throws SerializationException {
+            return SERIALIZER.serializeResponseAsString(ResponseMessage.build(requestID).result(message).create());
```

Comment:
Small nit. This is similar enough to returnSingleVertexResponse that you might want to make a generic function that can return the serialized string.

Reply:
Are you suggesting having 1 overloaded function where if you pass it a string it serializes a response around the string but if you don't give it anything it creates the response vertex and returns it?